Experiment automation/(LUMICKS) Reference kit automation/referencekit_automation.py

```diff
@@ -0,0 +1,335 @@
+
+    Parameters:
+    target: The target force to reach [pN]
+    match_threshold: The threshold for the match score to check whether the beads are still caught
+    speed: The speed at which to move the trap [um/s]
+    tolerance: The tolerance for the force measurement [pN]
+    tether_lost_threshold: The force threshold below which the tether is assumed lost [pN]"""
+
+    df = target - force.latest_value
+    throw_if_beads_lost(match_threshold)
+
+    while abs(df) > tolerance:  # um
+        bl.mirror1.move_by(dx=0.05 if df > 0 else -0.05, speed=speed)
+        df = target - get_force(dt=0.2)
+        throw_if_beads_lost(match_threshold)
+    check_tether_breakage(tether_lost_threshold)
+
+
+def get_force(dt=0.5):
+    t0 = bl.timeline.now
+    bl.pause(dt)
+    t1 = bl.timeline.now
+    return np.mean(force[t0:t1].data)
+
+
+def catch_dna(min_distance, max_distance, match_threshold, force_threshold, fishing_speed, fishing_attempts, dt, dna_length):
+    """Moves to the DNA channel and starts oscillating the trap until a prescribed force threshold is reached.
+
+    Parameters:
+    min_distance: The minimum distance to oscillate the trap to
+    max_distance: The maximum distance to oscillate the trap to
+    match_threshold: The threshold for the match score to check whether the beads are still caught
+    force_threshold: The force threshold to check whether we have caught DNA
+    fishing_speed: The speed at which to fish for DNA
+    fishing_attempts: The number of attempts to fish for DNA
+    dt: The time interval over which the force is averaged [s]
+    """
+    print("Moving to DNA channel")
+    bl.microstage.move_to(name_dna_channel)
+    bl.pause(2.0)
+    print("reset force")
+    bl.reset_force()
+    bl.pause(1)
+    f_current = get_force(dt=dt)
+    attempts = 0
+    while f_current < force_threshold:
+        if attempts > fishing_attempts:
+            raise RuntimeError(f"Max DNA fishing attempts {fishing_attempts} reached.")
+
+        print(f"Fishing for DNA: attempt {attempts}/{fishing_attempts}")
+        goto_distance(min_distance, match_threshold, speed=fishing_speed)
+        bl.pause(0.5)
+        goto_distance(max_distance, match_threshold, speed=fishing_speed)
+        f_current = get_force(dt=dt)
+
+        attempts += 1
+
+
+def make_kymograph(force_kymo, match_threshold, name, path):
+    goto_force(force_kymo, match_threshold)
+    try:
+        goto_force(force_kymo, match_threshold)
+        t0 = bl.timeline.now
+        bl.confocal.start_scan()
+        print("start kymograph")
+        bl.pause(10.0)
+        print(f"go to a force of {force_kymo} pN")
+        goto_force(force_kymo, match_threshold, speed=1)
+        bl.pause(kymo_duration)
+    finally:
+        print("Stop kymograph")
+        bl.confocal.abort_scan()
+        bl.pause(2)
+        t1 = bl.timeline.now
+        marker = bl.Marker(name, start = t0, stop = t1)
+        marker.export(Path(path))

```

Comment:
Don't really think this use of `Path` adds anything here. The suggestion on the last PR was to use Path to handle the concatenation, but since you chose to do that part by hand, I think it's better to just drop it entirely.
```suggestion
        marker.export(path)
```

Reply:
we got an error when running the script without adding `Path`